vm/src/obj/objfloat.rs

```diff
@@ -11,7 +11,7 @@ use crate::pyobject::{
     TypeProtocol,
 };
 use crate::vm::VirtualMachine;
-use hexf;
+use hexf_parse;
```

Comment:
Why was this required?

Reply:
`hexf` uses `proc_macro` at runtime (I think?) in order to do things like `hexf!("1e10")`, which wasn't supported for targeting redox because it needs to compile to a dylib and redox only supports staticlibs, so I found that since we don't use its macro we could just switch to `hexf_parse`.

Reply:
`hexf_parse` also makes more sense for our use-case.